packages/preview/cqut-thesis-typst/0.1.0/utils/theorem.typ

```diff
@@ -0,0 +1,165 @@
+#import "@preview/ctheorems:1.1.2": *
```

Comment:
ctheorems 1.1.2 reports deprecation warnings with Typst 0.12.0. It would be nice if you could update to 1.1.3, which should work better.